### DISCUSSION TITLE
Make un-submitted reviews/comments more obvious or visible

### DISCUSSION THREAD

**Discussion by alex-bradley** [community/community#16280] [Pull Requests]
I keep forgetting to go to the "Review Changes" dropdown to click "Submit review", so I've had comments sit there in "Pending" state for days.

I'll go through a PR, take some time to write thorough comments.  I'll get distracted by a slack message or something else important and when I come back I completely forgot that my comments haven't been "submitted"...and continue reviewing.

**Possible solutions**
- Notify me of pending reviews that I've started
- Use a different color button for pending reviews. I only just noticed that the green button at the top reads "Finish your review". Maybe make it orange instead?
- When I re-visit a page with a pending review, show something to remind me what state it's in. Maybe a popup?
- Show something on the main PR "Conversations" page to indicate I have a pending review. (Another related issue is being able to find my pending reviews...our team usually has 3 reviewers and there may be a lot of changes to go through.)

**Comment by y21** [comment]
+1, this just happened to me for the third time. First time I did a code review on Github I didn't even know you actually had to submit the review after creating one, so a suggested change in a review was sitting there for weeks until the PR author asked me to review, which had me confused until I saw the "pending" label on my review... Would be nice if it was more obvious that a review is still in the pending state and requires you to finally submit it before anyone can see it. Now that I think about it, I'm not even sure if I have more pending reviews because there's no way to find out (as far as I'm aware)

**Comment by alex-bradley** [comment]
Similar ask: https://github.com/community/community/discussions/10369
